### PR TITLE
Index

### PR DESCRIPTION
**Pull request by DarthDARKFather**
Created index static page, and wrote test for it. And added my name to the readme file.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  root "posts#index"
```
Comment (kenmazaika) [inline config/routes.rb]: This looks like it's missing the `PostsController` file, as well as `app/views/posts/index.html.erb`.

**Comment by kenmazaika** [comment]
Made one small suggestion.  Overall it looks good though!

File: config/routes.rb (hunk 1 above)
Comment (DarthDARKFather) [inline config/routes.rb]: I forgot to do a git add all be right back. Thanks for the suggestion cause I somehow missed that part.

Comment (kenmazaika) [inline config/routes.rb]: Perfect!  Make a comment after you push, otherwise, I won't be notified!

**Comment by DarthDARKFather** [comment]
Now it should be ready for merging.

File: config/routes.rb (hunk 1 above)
Comment (DarthDARKFather) [inline config/routes.rb]: Will do next time, I just pushed it with the changes.

File: spec/controllers/posts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+	describe "posts#index action" do
```
Comment (kenmazaika) [inline spec/controllers/posts_controller_spec.rb]: Nice test.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<h1 style="color:darkgreen;">Welcome to the Lets Pair Program!</h1>
```
Comment (kenmazaika) [inline app/views/posts/index.html.erb]: Eventually we should switch this out to define the styles in a file in `app/assets/stylesheets/` but we can do that later.

**Comment by kenmazaika** [comment]
:+1:

**Pull request merged**